src/config.ts

```diff
@@ -1072,3 +1076,29 @@ export const ALLOWED_CIE_TEST_FISCAL_CODES = pipe(
     return [] as ReadonlyArray<FiscalCode>;
   })
 );
+
+// UNIQUE EMAIL ENFORCEMENT variables
+
+export const FF_UNIQUE_EMAIL_ENFORCEMENT = pipe(
+  process.env.FF_IOLOGIN,
+  FeatureFlag.decode,
+  E.getOrElseW(() => FeatureFlagEnum.NONE)
+);
+
+export const UNIQUE_EMAIL_ENFORCEMENT_USERS = pipe(
+  process.env.UNIQUE_EMAIL_ENFORCEMENT_USERS,
+  // TODO: produce a ReadonlySet instead of ReadonlyArray
```

Comment:
You're right, very good suggestion! Can we create a task instead of commenting the code, so we don't miss it?

Reply:
is there a specific board for technical debt on `io-backend` ? should be resolvable by any stream working on `io-backend`